### ISSUE TITLE
Not working ESP32 Arduino Core 1.0 stable

### ISSUE THREAD

**Issue by ragulbalaji** [bitluni/ESP32CompositeVideo#7]

**Comment by marcel303** [comment]
Any output from the serial monitor?

**Comment by marcel303** [comment]
Try syncing to the latest version!

**Comment by ragulbalaji** [comment]
Can verify that CompositeVideo works, though CompositeVideoSimple bootloops...

**Comment by marcel303** [comment]
Yeah, I made a pull request to fix the second sketch too, but it's not in master yet. See https://github.com/bitluni/ESP32CompositeVideo/pull/12, or just use the code from CompositeVideo.
